orchagent/intfsorch.cpp

```diff
@@ -749,6 +749,10 @@ void IntfsOrch::doTask(Consumer &consumer)
             {
                 inband_type = value;
             }
+            else if (field == "vlan")
+            {
+                vlan = value;
```

Comment:
I don't see `vlan `defined

Reply:
Fixed as part of next commit. It was missed in local merge.